tests/Model/Domain/UpdateOpenTrackingResponseTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * Copyright (C) 2013 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+namespace Mailgun\Tests\Model\Domain;
+
+use Mailgun\Model\Domain\OpenTracking;
+use Mailgun\Model\Domain\UpdateOpenTrackingResponse;
+use Mailgun\Tests\Model\BaseModelTest;
+
+class UpdateOpenTrackingResponseTest extends BaseModelTest
+{
+    public function testCreate()
+    {
+        $json =
+            <<<'JSON'
+{
+  "open": {
+    "active": "no"
```

Comment:
As mentioned in the other file, it seems to be a boolean as part of the API response

```suggestion
    "active": false
```